src/app/login-form/login-form.component.html

```diff
@@ -1,8 +1,21 @@
-<form class="login-form">
-  <md-input-container class="full-width">
-    <input mdInput placeholder="User name"/>
-  </md-input-container>
-  <md-input-container class="full-width">
-    <input mdInput placeholder="Password"/>
-  </md-input-container>
-</form>
+<div class="mat-input-container">
+
+  <form>
+
+    <div class="mat-row">
+      <md-input-container>
+        <input mdInput placeholder="User name"/>
+      </md-input-container>
+    </div>
+
+    <div class="mat-row">
+      <md-input-container>
+        <input mdInput placeholder="Password"/>
+      </md-input-container>
+    </div>
+
+    <button md-raised-button color="primary">Login</button>
```

Comment:
Put the button in a div `mat-row`. If there is an password error, the Login button is too close to the error,
![image](https://user-images.githubusercontent.com/14135364/29758793-7d196aa4-8be0-11e7-8e45-c19d805a75b9.png)

Reply:
The font of label and input are different to `Login`, please help make it consistent